modules/users/client/controllers/admin/adduser.client.controller.js

```diff
@@ -0,0 +1,92 @@
+(function () {
```

Comment:
Also, naming convention for files in the mean.js repo is snake case for joined words and dot case for the rest of the "namespace"
ex.
add-user.client.controller.js
instead of
adduser.client.controller.js